### PR TITLE
Render tokenId for erc1155

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

File: src/shared/claim-page.tsx [hunk 1]
```diff
@@ -4,24 +4,31 @@ import {
...
 
 export const ClaimPage: React.FC<React.PropsWithChildren<ClaimPageProps>> = ({
   contract,
-
+  tokenId,
   children,
 }) => {
   const { data: metadata, isLoading } = useContractMetadata(contract);
+  const { data: nft } = useNFT(contract, tokenId);
+
```
Comment (jnsdls) [inline src/shared/claim-page.tsx:31]: I would like to split this into 2 again now, `TokenClaimPage` (with the token) and `ContractClaimPage` (without the token)

we shouldn't even hav useNFT in the contract claim page case and we might not need contract metadata in the token claim page case? might just want to render token name and description instead?

File: src/shared/claim-page.tsx [hunk 2]
```diff
@@ -35,14 +42,10 @@ export const ClaimPage: React.FC<React.PropsWithChildren<ClaimPageProps>> = ({
...
-                objectFit="contain"
-                w="100%"
-                h="100%"
-                src={metadata?.image}
-                alt={metadata?.name}
-              />
+            {tokenId && nft?.metadata ? (
+              <ThirdwebNftMedia metadata={nft.metadata} />
+            ) : metadata?.image ? (
+              <ThirdwebNftMedia metadata={metadata} />
```
Comment (jnsdls) [inline src/shared/claim-page.tsx:48]: yeah ideally we split these high level into 2 components, one for erc20 and erc721 and one for erc1155

**Review by jnsdls — CHANGES_REQUESTED** [review]
I'd like to handle this more explicitly I think

**Comment by nachoiacovino** [comment]
Superseded by #73

**Pull request closed without merging**